docs/Technical Reference.md

```diff
@@ -223,41 +223,15 @@ In the row of buttons at the top of the pane, select `Modify Access Policy`. Add
 
 Select `Submit` and your search domain should now be open to the public.
 
-### Federations and bucket config
+### Use Google to sign into Quilt
 
-In this section we will discuss how you can configure your catalog instance using _federations_ and _bucket config_.
+You can configure your Quilt stack to authenticate with Google. Follow these [instructions](https://developers.google.com/identity/protocols/OAuth2UserAgent) to (1) enable Gmail APIs and (2) create authorization credentials to identify your Quilt stack to Google's OAuth 2.0 server.
```

Comment:
please break these lines at 80 characters (one newline becomes space when the paragraph is rendered).
otherwise these sentences are very clumsy to edit.

Reply:
```suggestion
You can enable users on your Google domain to sign in to Quilt. Refer to [Google's instructions on OAuth2 user agents](https://developers.google.com/identity/protocols/OAuth2UserAgent) and complete the following:

1. Enable Gmail APIs
1. Create authorization credentials to identify your Quilt stack to Google's OAuth 2.0 server.
```

01234567890123456789012345678901234567890123456789012345678901234567890123456789